tests/services/mapping_api_service.py

```
@@ -0,0 +1,91 @@
+        response_datas = response.json()
+        logger.info(f"Mapping status: {response_datas[0]['general_status']}")
+        if is_verbose:
+            for mapping_results in response_datas:
+                logger.info(f"Mapping job id: {mapping_results['job_id']}")
+                logger.info(f"Mapping status: {mapping_results['general_status']}")
+                logger.info(f"Mapping ready count: {mapping_results['ready_count']}")
+                logger.info(f"Mapping total count: {mapping_results['total_count']}")
+                logger.info("")
+        return response_datas
+    else:
+        logger.info(f"Failed to get mapping results")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)